apps/spruce/src/pages/image/PackagesTable/PackagesTable.test.tsx

```diff
@@ -0,0 +1,315 @@
+import { MockedProvider } from "@apollo/client/testing";
+import { RenderFakeToastContext } from "context/toast/__mocks__";
+import {
+  ImagePackagesQuery,
+  ImagePackagesQueryVariables,
+} from "gql/generated/types";
+import { IMAGE_PACKAGES } from "gql/queries";
+import {
+  renderWithRouterMatch as render,
+  screen,
+  waitFor,
+  within,
+  userEvent,
+} from "test_utils";
+import { ApolloMock } from "types/gql";
+import { PackagesTable } from ".";
+
+describe("packages table", () => {
+  it("shows name field data", async () => {
+    const { Component } = RenderFakeToastContext(
+      <MockedProvider mocks={[imagePackagesPageOneMock]}>
+        <PackagesTable />
+      </MockedProvider>,
```

Comment:
instead of redeclaring this in every test, can we have a `wrapper` component that gets passed to the `render` function? Similar to what you did in the tests for the Event Log